server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -331,8 +294,6 @@ public static Builder builder()
     private static final boolean DEFAULT_EMIT_BALANCING_STATS = false;
     private static final boolean DEFAULT_KILL_ALL_DATA_SOURCES = false;
     private static final int DEFAULT_MAX_SEGMENTS_IN_NODE_LOADING_QUEUE = 0;
-    private static final double DEFAULT_COMPACTION_TASK_RATIO = 0.1;
-    private static final int DEFAILT_MAX_COMPACTION_TASK_SLOTS = Integer.MAX_VALUE;
 
     private Long millisToWaitBeforeDeleting;
```

Comment:
Please annotate properties and parameters in this class `@Nullable` as appropriate.

Reply:
Added.